### PR TITLE
log AD Sync Tasks

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

**Comment by domdinicola** [comment]
[ch7180]

File: src/etools/libraries/azure_graph_api/tasks.py [hunk 1]
```diff
@@ -28,28 +32,46 @@ def sync_user(username):
...
     logger.info('Azure Delta Sync Process started')
-    url = cache.get(
-        AZURE_GRAPH_API_USER_CACHE_KEY,
-        '{}/{}/users/delta?$top={}'.format(
-            settings.AZURE_GRAPH_API_BASE_URL,
-            settings.AZURE_GRAPH_API_VERSION,
-            settings.AZURE_GRAPH_API_PAGE_SIZE
+    log = VisionSyncLog(country=Country.objects.get(schema_name="public"), handler_name='UserADSyncDelta')
+    try:
+        url = cache.get(
```
Comment (reinbach) [inline src/etools/libraries/azure_graph_api/tasks.py]: How come we cache this `url` and not in `sync_all_users`? maybe move this `url` param into a central location that both of these functions get/set the url?

Actually the only difference besides the usage of cache, appears to be the `handler_name` parameter passed to `VisionSyncLog`

Comment (domdinicola) [reply src/etools/libraries/azure_graph_api/tasks.py]: the urls are slightly different there's the '/delta' for the delta task

Comment (reinbach) [reply src/etools/libraries/azure_graph_api/tasks.py]: ah missed that

**Pull request merged**